index.html

```diff
@@ -0,0 +1,106 @@
+		</div>
+		<h2 class="h_contacts">Contacts:</h2>
+		<a href="tel:+35283318635">
+			<div class="phone"></div>
+		</a>
+		<a href="mailto:#@yandex.ru">
+			<div class="mail"></div>
+		</a>
+		<a href="skype:#">
+			<div class="skype"></div>
+		</a>
+		<a href="tg://resolve?domain=#">
+			<div class="telegram"></div>
+		</a>
+		<a href="https://www.linkedin.com/">
+			<div class="linkedin"></div>
+		</a>
+	</header>
+	<div class="info">
+		<h2 class="name">Victor <span>Mitskevich</span></h2>
+		<p class="by_line">Front-end developer</p>
+		<div class="secondary_heading ">
+			<img src="assets/img/skills.png" alt="hammer">
+			<div class="line">
+				<h2>Skills:</h2>
+			</div>
+		</div>
+		<div class="skills_info content_info">
+			<ul>
+				<li>HTML;</li>
+				<li>CSS;</li>
+				<li>Javascript;</li>
+				<li>Vector graphics(CorelDraw).</li>
+			</ul>
+		</div>
+		<div class="secondary_heading">
+			<img src="assets/img/code-ex.png" alt="work">
+			<div class="line">
+				<h2>Code examples:</h2>
+			</div>
+		</div>
+		<div class="code_info content_info"><a href="https://github.com/MitskevichVS"><img
+				src="assets/img/git.png" alt="github">Github link.</a></div>
+		<div class="secondary_heading">
+			<img src="assets/img/expirience.png" alt="mountain">
+			<div class="line">
+				<h2>Expirience:</h2>
+			</div>
+		</div>
+		<div class="expirience_info content_info">
+			<p>Rolling Scopes school;</p>
+		</div>
+		<div class="secondary_heading">
+			<img src="assets/img/education.png" alt="education">
+			<div class="line">
+				<h2>Education:</h2>
+			</div>
+		</div>
+		<div class="education_info content_info">
+			<ul>
+				<li>Bachelor degree: Belorusian State University of Informatics and Radioelectronics, Radioelectronics facility, Radioelectonics systems.</li>
+				<li>Masters degree: Belorusian State University of Informatics and Radioelectronics, Radioelectonics faculty, Radio engineering, including systems and devices for radio navigation, location and television.</li>
+			</ul>
+		</div>
+		<div class="secondary_heading">
+			<img src="assets/img/english.png" alt="international">
+			<div class="line">
+				<h2>English level:</h2>
+			</div>
+		</div>
+		<div class="english_info content_info">
+			<p>Pre-intermediate.</p>
+		</div>
+		<div class="secondary_heading">
+			<img src="assets/img/ok.png" alt="summary">
+			<div class="line">
+				<h2>Summary:</h2>
+			</div>
+		</div>
+		<div class="summary_info content_info">
+			<p>My main goal at the moment is to learn newest technologies and tools for working as a developer. As a junior web developer i’m ready to compete, work overtime, work on mistakes and learn from expirience.</p>
+		</div>
+	</div>
+	<footer>
+		<p class="footer_info">Mitskevich &copy; 2019<br>
+			<a href="tel:+35283318635">Contacts: +352 83 331 86 35</a><br>
+			<a href="mailto:#@yandex.com">Mail: #@yandex.com</a>
+		</p>
+	</footer>
```

Comment:
footer doesn't look like footer at all. In your case it just a part of the sidebar